collection_json.py

```diff
@@ -6,6 +6,59 @@
 __version__ = '0.1.0'
 
 
+class ArrayProperty(object):
+
+    """Property which converts from any enumerable to a typed Array instance."""
```

Comment:
I'd change this to somtehing like  'A descriptor that converts any enumerable to a type Array instance.' to me it clear this class implements the descriptor protocol